docs/en/reference/configuration.rst

```diff
@@ -296,6 +296,8 @@ pdo\_oci / oci8
    add the INSTANCE_NAME parameter in the connection. It is generally used
    to connect to an Oracle RAC server to select the name of a particular instance.
 
+Complete connection descriptors are also supported. You only need to provide the following parameter, and none of the above:
```

Comment:
This breaks the list in the markup. Please start with `- connectstring (string)` (as with the other parameters) and then have the complete description follow after. Also the statement is not quite true. `user` and `password` are still required, right? Please consider that in the description.